app/config/settings.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * This file returns a set of configurations for the respective environment that the app is running in.
+ */
+
+const settings = {
+  dev: {
+    baseURL: "https://ill-gold-parrot-sock.cyclic.app",
+  },
+  prod: {
+    // update this URL once we deploy to prod env
+    baseURL: "https://some-url-in-prod.com",
+  },
+};
+
+const getCurrentSettings = () => {
+  if (__DEV__) return settings.dev;
```

Comment:
Where will we store this __DEV__ does it mean from environment variable? Or from webpack?

Reply:
__DEV__ is a global variable from react native packager. It is set during runtime by react native and is available to use anywhere in the app for encapsulating environment specific code blocks.